web/vite.config.ts

```diff
@@ -10,6 +10,18 @@ export default defineConfig(() => ({
     svelte(),
     nodeResolve({
       extensions: ['.js', '.ts']
-    })
+    }),
+    {
+      name: 'html-transform',
+      enforce: 'pre' as const,
```

Comment:
To fix ESLint warning call:

```
pnpm eslint --fix .
```

Or change the code according to the error text.